importer/archive_test.go

```diff
@@ -37,12 +38,44 @@ func TestArchive(t *testing.T) {
 		})
 	})
 
-	Convey("Given a valid zip file", t, func() {
+	Convey("Given an invalid zip file (no index.html)", t, func() {
```

Comment:
just looking at this and thought 😄  are we are being too specific looking for `index.html`? maybe we make it just `.html` - 1 html file allowed

Reply:
sticking to index.html .. lets re visit if we need to